core/components/Notification.vue

```diff
@@ -27,6 +27,11 @@ export default {
       switch (action) {
         case 'close':
           this.notifications.splice(id, 1)
+          break
+        case 'goToCheckout':
+          this.$router.push('/checkout')
+          this.notifications.splice(id, 1)
+          break
```

Comment:
To make the action mechanism more generic please EventBus.$emit, the eventName should be set to “action”. By doing so, extension developers will be able to handle their own custom cart actions. 

Then the handler of goToAction should’ve been handled by some event listener bound in the Microcart.vue in the core or the default theme

We shouldn’t have placed any specific logic inside general component of Notification; this components shouldn’t have known that we have some checkout in the app etc just display the general notifications and emit general events

The event name should be: “cart-after-goToCheckout”